src/Blazor.Gitter.Library/Services/GitterApi.cs

```diff
@@ -55,9 +54,9 @@ public Task<IChatUser> GetChatUser(string UserId)
             throw new NotImplementedException();
         }
 
-        public Task<IEnumerable<IChatUser>> GetChatRoomUsers(string RoomId)
+        public async Task<IEnumerable<IChatUser>> GetChatRoomUsers(string RoomId, IRoomUserOptions Options)
         {
-            throw new NotImplementedException();
+            return await HttpClient.GetJsonAsync<GitterUser[]>($"{APIROOMS}/{RoomId}/users{Options}");
```

Comment:
This needs to be the new GetFromJsonAsync to merge correctly into master now that your other PRs are merged.
I can do it, but if you do it it will definitely keep you as the author.

Reply:
Yup. I tried to keep the PRs independent from each other.

Will do it later this week.

Reply:
ok thanks